image_classification/requirements.txt

```diff
@@ -0,0 +1,5 @@
+--extra-index-url https://google-coral.github.io/py-repo/
```

Comment:
Split the requirements to 2 files similar to this:
https://github.com/tensorflow/examples/commit/615c1a76e76262231d23fea7e54edcdc97bbf9ff

There was an issue with PyPi that we no longer can have one requirements.txt file.